jobs/furnishings/src/furnishings-templates/gazette-notice.xml

```diff
@@ -18,7 +18,13 @@
                 {%- else -%}
                 <corp_num>{{ f.business_identifier[2:] }}</corp_num>
                 {%- endif %}
-                <text>{{ f.business_name }}</text>
+                <text>
+                    {% if furnishing['category'] == 'DISSOLUTIONS' -%}
```

Comment:
One small thing here, I understand we also need the time for REGISTRATIONS CANCELLED. However, I don't see them in the code or implemented yet? I can add the conditional here but it'll be useless basically. Thoughts?

Reply:
The REGISTRATIONS CANCELLED functionality was added in #2948, you probably just need to rebase to see the implementation. Can you double check with Dylan if the time for D3 EPs is the same as for D3 BC companies (ie. the time the job was ran at)?

Reply:
Thanks for pointing this out Leo.

So, I've confirmed with Dylan. The process for REGISTRATIONS CANCELLED is going to follow the same as  DISSOLUTIONS.

I've rebased and added it to the XML template.